lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -969,7 +489,7 @@ def main():
             access_logs_enabled=dict(type='bool'),
             access_logs_s3_bucket=dict(type='str'),
             access_logs_s3_prefix=dict(type='str'),
-            deletion_protection=dict(default=False, type='bool'),
+            deletion_protection=dict(type='bool'),
```

Comment:
This is documented to default to False. Is that a bug? Should update the docs too.